charts/kubewarden-controller/questions.yaml

```
@@ -1,60 +1,104 @@
-  show_if: auditScanner.enable=true
-  label: Enable Policy Reporter
-  description: |
-    Policy reporter is a UI to visualize the reports generated by Background
-    Audit checks
-  group: "Features"
-- variable: tls.source
-  default: "cert-manager-self-signed"
-  description: "Source for TLS certificates"
-  label: TLS certificates source
-  type: enum
-  options:
-  - "cert-manager-self-signed"
-  - "cert-manager"
-  show_subquestion_if: "cert-manager"
-  group: "Certificates"
-  subquestions:
-  - variable: "tls.certManagerIssuerName"
+  - variable: "auditScanner.enable"
+    type: boolean
+    default: false
+    required: true
+    label: Enable Background Audit check
+    description: |
+      The Background Audit Check looks at existing Kubernetes resources and
+      periodically determines whether they are still compliant with regards to the
+      policies that are currently defined.
+    group: "Features"
+  - variable: "auditScanner.cronJob.schedule"
+    type: string
+    default: "*/60 * * * *"
+    show_if: auditScanner.enable=true
+    label: Schedule
+    description: |
+      Schedule of the Background Audit check. Follows the Unix-cron syntax.
+    group: "Features"
+  - variable: "auditScanner.serviceAccountName"
     type: string
-    default: ""
-    label: cert-manager Issuer name
+    default: "audit-scanner"
+    show_if: auditScanner.enable=true
+    label: ServiceAccount name
     description: |
-      Name of cert-manager Issuer
+      The default audit-scanner ServiceAccount is bound to the ClusterRoles
+      "view" (allows read-only access to most objects in a namespace, does not
+      allow viewing secrets, roles or role bindings), and "audit-scanner-cluster-role"
+      (allows read-write to Kubewarden resources and PolicyReports).
+    group: "Features"
+  - variable: "auditScanner.policyReporter"
+    type: boolean
+    default: false
+    show_if: auditScanner.enable=true
+    label: Enable Policy Reporter
+    description: |
+      Policy reporter is a UI to visualize the reports generated by Background
+      Audit checks
+    group: "Features"
+  - variable: tls.source
+    default: "cert-manager-self-signed"
+    description: "Source for TLS certificates"
+    label: TLS certificates source
+    type: enum
+    options:
+      - "cert-manager-self-signed"
+      - "cert-manager"
+    show_subquestion_if: "cert-manager"
     group: "Certificates"
-    show_if: "tls.source=cert-manager"
+    subquestions:
+      - variable: "tls.certManagerIssuerName"
+        type: string
+        default: ""
+        label: cert-manager Issuer name
+        description: |
+          Name of cert-manager Issuer
+        group: "Certificates"
+        show_if: "tls.source=cert-manager"
+  - variable: "telemetry.metrics.enabled"
+    type: boolean
+    default: false
+    required: true
+    label: Enable Metrics
+    description: |
+      Enable metrics collection for all Policy Servers and the Kubewarden Controller
```

```suggestion
      Enable metrics collection for all Policy Servers and the Kubewarden Controller.
```

The multiline description string gets rendered as single line. Add dots for readability.